### PR TITLE
Fixed minor punctuation and grammatical errors

### PR DESCRIPTION
**Pull request by iamakshat01**
## Type of change

- [ ] Fixed punctuation errors in doc

## Checklist:

- [Yes] I have signed the Google CLA <https://cla.developers.google.com>

### CODE REVIEW

File: docs/tutorials/network-and-buffering-config.md [hunk 1]
```diff
@@ -102,8 +102,8 @@ impose several restrictions on the content that a webpage has access to.
 
 One restriction is [CORS][] (Cross-Origin Resource Sharing).  This requires
 network requests to be made to the same origin, or for the server to explicitly
-give access.  An "origin" refers to the domain name (e.g. `api.example.com`),
-the scheme (e.g. `https:`), and the port (e.g. 80).  If you host your assets on
+give access.  An "origin" refers to the domain name (e.g., `api.example.com`),
```
Comment (theodab) [inline docs/tutorials/network-and-buffering-config.md:105]: I hadn't seen this particular styling often before, so I looked it up. According to [some grammar sources](https://jakubmarian.com/comma-after-i-e-and-e-g/) this is apparently the way that American style guides recommend, and British style guides recommend no comma. Well, we've been going with American spelling and grammar styles in other places, so I guess we should use this.
Learn something new every day, huh.

**Comment by theodab** [comment]
Thank you for your contribution!

**Comment by iamakshat01** [comment]
Hi theodab, I am a beginner in open source world. Looking forward to do contributions to this project. Can you suggest me some beginner-friendly issues  that I can work on?

**Comment by nbcl** [comment]
Hello @iamakshat01!

That is great to hear!

I think the best way to start is having a look under the issues labeled ["easy?"](https://github.com/google/shaka-player/issues?q=is%3Aissue+is%3Aopen+label%3Aeasy%3F), as they could probably be the easier ones to approach.

Apart from that, I would also recommend checking out the [Open Source Guides](https://opensource.guide/) collection, maintained by GitHub, as they offer some great advice to begin working on the open source world.

Hope you find what you are looking for, and let me know if you need any more advice, I will be happy to help!

**Comment by iamakshat01** [comment]
Ok, I will go try with "easy?" labeled issues. Is there any community channel for shaka player?

**Pull request merged**